cdap-api/src/main/java/co/cask/cdap/api/data/batch/TaggedInputSplit.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.data.batch;
+
+/**
+ * Tags an InputSplit with a name.
+ */
+public interface TaggedInputSplit {
```

Comment:
I don't think we need this class in api. See my comment about how to expose the input name at 

https://github.com/caskdata/cdap/pull/5276/files#r55805739